src/components/ BooksForm.js

```diff
@@ -0,0 +1,17 @@
+const BooksForm = () => {
```

Comment:
- [ ] Same here. Since `BooksForm` is going to connect to the redux store in the next milestone to add a book to redux, we need to move it to `src/containers` directory.